make.ps1

```diff
@@ -0,0 +1,117 @@
+# Run the command below to add make facade commands to powershell: 
```

Comment:
Isnt there a native make binary for windows?

Reply:
as discussed, thanks for pointing me to https://chocolatey.org/packages/make. This definitely seems to be the way to go to keep things simple but will require re-writing Makefiles with cross-platform in mind